### ISSUE TITLE
Gun Script Needs Verified

### ISSUE THREAD

**Issue by iShauny** [benzoferrari123/pinerf#1]
Once main turret base script is finished, gun script needs tested with main system

**Comment by iShauny** [comment]
Including gun_controller now

**Comment by iShauny** [comment]
any progress on turret base script @benzoferrari123 ?

**Comment by iShauny** [comment]
Purely software verification is complete, and another issue exists around unit tests. So I will resolve this for now as the script appears to be working as intended.
